### PR TITLE
[env](compile)open compile check in pipeline op 

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

File: be/src/pipeline/exec/join/join_op.h [hunk 1]
```diff
@@ -43,7 +44,7 @@ namespace doris::pipeline {
  *  RowRefListWithFlags is a list of many RowRefWithFlags. This means each row will have different visited flags. It's used for join operation which has `other_conjuncts`.
  */
 struct RowRef {
-    uint32_t row_num = 0;
+    size_t row_num = 0;
```
Comment (HappenLee) [inline be/src/pipeline/exec/join/join_op.h:47]: The change would affect performance; it cannot be done

File: be/src/pipeline/exec/table_function_operator.cpp [hunk 2]
```diff
@@ -288,7 +290,7 @@ Status TableFunctionOperatorX::open(doris::RuntimeState* state) {
         }
     }
 
-    for (size_t i = 0; i < _child_slots.size(); i++) {
```
Comment (HappenLee) [inline be/src/pipeline/exec/table_function_operator.cpp:291]: the `_child_slot.size()` is `size_t` why not change to `int`

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Pull request closed without merging**